recipes/readabilipy/recipe.yaml

```diff
@@ -0,0 +1,43 @@
+context:
+  version: 0.3.0
+
+package:
+  name: readabilipy
+  version: ${{ version }}
+
+source:
+- url: https://files.pythonhosted.org/packages/b8/e4/260a202516886c2e0cc6e6ae96d1f491792d829098886d9529a2439fbe8e/readabilipy-0.3.0.tar.gz
```

Comment:
Please use the canonical url:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example-v1/recipe.yaml#L23